src/dialogs/prompt/template.js

```diff
@@ -1,5 +1,8 @@
 module.exports = ({ pallette, message, defaultValue, buttons, id }) => (`<!DOCTYPE html>
 <html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
```

Comment:
There is a shorter notation in HTML5: 
```suggestion
    <meta charset="utf-8" /> 
```

Reply:
Clearly I'm old 😊  Thanks for pointing that out!